src/server/upgrade.rs

```diff
@@ -89,6 +90,36 @@ fn interpret_options(options: &Upgrade) -> anyhow::Result<ToDo> {
 
 pub fn upgrade(options: &Upgrade) -> anyhow::Result<()> {
     let todo = interpret_options(&options)?;
+    if let ToDo::InstanceUpgrade(name, version) = &todo {
+        let project_dirs = destroy::find_project_dirs(name)?;
+        if !project_dirs.is_empty() {
+            destroy::print_instance_in_use_warning(name, &project_dirs);
+            let current_project = project::project_dir_opt(None)?;
+            eprintln!("To continue with the upgrade, run:");
```

Comment:
I think this message should be changed when force is active. Because writing `To continue with update` and then actually doing the upgrade is confusing.